wordpress-mobile/WordPress-Android
# Update Prologue from settings

Fixes #14949

Related FluxC PR - https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/2038

This PR introduces 2 changes:
- the prologue is shown from settings only before blogging reminders get set the first time
- the prologue from settings copy is updated 
both of these changes are coming from the updated designs

To test:
- Clear app data and make sure the blogging reminders flag is turned on
- Go to settings/Blogging reminders
- Notice the prologue is shown with the new copy
- set blogging reminders
- Go to settings/Blogging reminders again
- Notice the prologue is skipped and the day selection is shown

## Regression Notes
1. Potential unintended areas of impact
- None

PR submission checklist:
